### PR TITLE
[WebApp] Fix search sort value revert issue

### PR DESCRIPTION
**Pull request by rners01**
https://www.notion.so/saladtech/WebApp-Search-sort-value-label-revert-issue-073712e0a784486fbdaddbcec51d336d

### CODE REVIEW

File: packages/web-app/src/components/Dropdown.tsx [hunk 1]
```diff
@@ -16,45 +15,52 @@ const styles = (theme: SaladTheme) => ({
 
 interface Props extends WithStyles<typeof styles> {
   options?: OptionsOrGroups<any, any>
```
Comment (vitto-moz) [inline packages/web-app/src/components/Dropdown.tsx:17]: It's unclear what the option type should be, could we use any explicit type?

File: packages/web-app/src/components/Dropdown.tsx [hunk 2]
```diff
@@ -16,45 +15,52 @@ const styles = (theme: SaladTheme) => ({
...
-      />
-    )
+      return { ...provided, opacity, transition, color }
+    },
   }
+
+  return (
+    <Select
+      className={classes.container}
+      value={selectedOption}
```
Comment (vitto-moz) [inline packages/web-app/src/components/Dropdown.tsx:45]: Can we align naming to 
```suggestion
      value={selectedValue}
```

File: packages/web-app/src/components/Dropdown.tsx [hunk 3]
```diff
@@ -16,45 +15,52 @@ const styles = (theme: SaladTheme) => ({
...
   onChange?: (value?: any) => void
 }
 
-const customStyles = {
-  singleValue: (provided: any, state: any) => {
-    const opacity = state.isDisabled ? 0.5 : 1
-    const transition = 'opacity 300ms'
-    const color = '#DBF1C1'
+const _Dropdown: FC<Props> = ({ classes, options, value, onChange }) => {
+  const selectedValue = value
```
Comment (vitto-moz) [inline packages/web-app/src/components/Dropdown.tsx:23]: what is the purpose of this variable set

File: packages/web-app/src/components/Dropdown.tsx (hunk 1 above)
Comment (rners01) [reply packages/web-app/src/components/Dropdown.tsx]: added

File: packages/web-app/src/components/Dropdown.tsx (hunk 3 above)
Comment (rners01) [reply packages/web-app/src/components/Dropdown.tsx]: refactored

**Review by vitto-moz — APPROVED** [review]

**Review by Maks19 — APPROVED** [review]

**Pull request merged**